### PR TITLE
Clann morna

### PR DESCRIPTION
**Pull request by cyocum**
I have created the file and linked it to the IrishGen database.  Please have a look over it and let me know if I have misunderstood anything  There are several questions that I have left in the comments so please let me know what you think.  Do not worry about failing google build.  I will resolve that as soon as you have approved the cloud build PR.

### CODE REVIEW

**Comment by Eystein-Thanisch** [comment]
Here are my suggested changes for the Clann Morna tract. I've added comments explaining myself where I think this is needed. Please have a look and let me know what you think. This is a difficult, obscure, and, I am convinced, not entirely coherent text!

Two other things. First, it is currently failing its checks but I can't work out why. I've read the file manually twice and also run Apache Jena on it. Secondly, should this really be in irish-narr and not irishGen? It isn't really a single narrative and the interest is surely genealogical?

**Comment by cyocum** [comment]
Great.  Thanks. If you have resolved a question in the file and in a comment, it is probably best to remove any comments and leave the corrected version in the file. The build problem will be resolved after you have rebased and merged so do not worry about that.  As for the repository it should be in, I think that we should leave it here for now.  If things change, we can move it without any trouble.  I think this is ready to merge so feel free to do so.

**Comment by Eystein-Thanisch** [comment]
Unfortunately, a lot of the questions in the comments remain unresolved and, even where I have attempted to resolve them, I am not very confident of my solution! Ancient Connacht is all very uncertain territory, historiographically. Either we should both agree on an interpretation of the problematic sections or we should keep the comments in the file so that other users understand our reasoning. 

Also, you might have noticed that a lot of the Gamanraid characters from this text also come up in _Ferchuitred Medba_ (I didn't realise this when I proposed adding this text - indeed, I wasn't thinking about _Ferchuitred Medba_ at all). So they'll be on the front line when we generate contextual data for that text and we need to make sure that our encoding of them is as sound as possible.

Can I rebase with a pull request open? If so, I'll do that, then at least clear it with the Build!

**Comment by cyocum** [comment]
Ok. Sounds good. You can rebase and merge to master then push to origin as
long as you do it locally on your machine. The system should close the PR
automatically.

**Comment by Eystein-Thanisch** [comment]
Sorry, I was still updating my previous comment when you replied!

**Comment by Eystein-Thanisch** [comment]
Sorry, I was still updating my previous comment when you replied :-P We need to make sure this file is as sound as possible, as it will feature prominently when we generate the data for Abi. Would you be able to look through the comments and see you agree or can improve on my interpretations?

**Comment by cyocum** [comment]
Ok. I cannot do that today but I may be able to do it tonight and
definitely tomorrow.

**Comment by Eystein-Thanisch** [comment]
That would be most reassuring. Thanks!

**Comment by Eystein-Thanisch** [comment]
So can I rebase but not merge (yet) and still keep the PR open?

**Comment by cyocum** [comment]
If you force push, yes.

File: Lec/clann_morna.trig [hunk 1]
```diff
@@ -406,7 +411,7 @@ _:missing-d23e02b6
 # to be gained from trying to tease apart two characters from this. EPT
 
 <#Forco>
-        a foaf:Person ;
+        a irishTitles:Rí ;
```
Comment (Eystein-Thanisch) [inline Lec/clann_morna.trig:414]: I think we might have diverged in how we use `irishTitles:Rí`. I've been using it only in the context of regnal lists, where there is also succession data, whereas you are using it for everyone who is stated to be a king, even in isolation. Your use is probably better, but it does mean there will be quite a few kings scattered about only identified by `foaf:title`...

**Review by Eystein-Thanisch — APPROVED** [review]

**Comment by Eystein-Thanisch** [comment]
And the checks have been passed!

**Pull request merged**